src/components/token/token.js

```diff
@@ -70,14 +69,28 @@ export const EuiToken = ({
 };
 
 EuiToken.propTypes = {
-  iconType: PropTypes.string.isRequired,
-  color: PropTypes.string.isRequired,
-  isSolidColor: PropTypes.bool,
-  hideBorder: PropTypes.bool,
```

Comment:
Did not see this in use anywhere in your mapping/css so I removed it. We can always add it back in if you wanted to add a "no-border" option. Might be a good idea!

Reply:
Unless there's objections, I'd like to throw that `no-border` option back in. The thought here was that by using the `rectangle` shape and a `hideBorder` flag you could very easily use this component in the instance that @cchaos mentioned 👇

![](https://user-images.githubusercontent.com/549577/47726226-6b355b80-dc30-11e8-8fe3-bf0d110d0fa4.png)

Reply:
Makes sense to me. Like I said, I only removed it because the prop wasn't being used.